.flowconfig

```diff
@@ -27,4 +27,4 @@ untyped-import
 untyped-type-import
 
 [version]
-0.97.0
\ No newline at end of file
+>=0.97.0
```

Comment:
Flow doesn't follow semver, so I don't think we should do this. What happened that led to this?

Reply:
I'm running 0.98.0, so I changed it.
Is it likely to have breaking changes? As I need the latest flow version for some react projects I have.

Reply:
Perhaps we could just update it to the latest version and pin it to that one?